MaxText/pyconfig.py

```diff
@@ -193,6 +193,7 @@ def validate_and_assign_remat_tensors(keys):
   # list of allowed tensors for custom remat policy
   tensors = [
       "decoder_layer_input",
+      "context",
```

Comment:
We should add a comment somewhere in maxtext pointing to the context tensor in TE codebase, but doesn't need to be this PR

Reply:
E.g. here https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/jax/attention.py#L581-L583

Also we should add context to "custom" remat policy option, doesn't need to be this PR

Reply:
Thanks Matt. I'll add this link as a comment in base.yml

For adding context to "custom" remat, are you meaning something different than adding the tensor here + in base.yml?